dynamic_rest/fields/generic.py

```diff
@@ -84,7 +84,7 @@ def to_representation(self, instance):
             # We want the pk to be represented as an object with type,
             # rather than just the ID.
             pk_value = self.get_pk_object(
-                serializer_class().get_plural_name(),
+                serializer_class().get_name(),
```

Comment:
Oh, looks like the [place I'm using](https://github.com/AltSchool/dynamic-rest/pull/95/files#diff-f072b5566ec48c2be4de15dc12a59eacR150) it I'm already invoking it as a class method